docs/products/kafka/kafka-connect/concepts/list-of-connector-plugins.rst

```diff
@@ -0,0 +1,104 @@
+List of available Apache Kafka Connect connectors
+=================================================
+
+The following Apache Kafka Connect connectors are currently available, and can
+be used in any Aiven for Kafka services with Kafka Connect enabled. 
+
+
+Source connectors
+-----------------
+
+Source connectors enable the integration of data from an existing technology into an Apache Kafka topic. The following is the list of available connectors:
+
+* `Couchbase <https://github.com/couchbase/kafka-connect-couchbase>`__
+
+* `Official MongoDB <https://docs.mongodb.com/kafka-connector/current/>`__ :badge:`preview,cls=badge-secondary badge-pill`
+
+* `Debezium for MongoDB <https://debezium.io/docs/connectors/mongodb/>`__
+
+* `Debezium for MySQL <https://debezium.io/docs/connectors/mysql/>`__
+
+* `Debezium for PostgreSQL <https://help.aiven.io/kafka/setting-up-debezium-with-aiven-postgresql>`__
+
+* `Debezium for SQL Server <https://debezium.io/docs/connectors/sqlserver/>`__
+
+* `Google Cloud Pub/Sub <https://github.com/GoogleCloudPlatform/pubsub/tree/master/kafka-connector>`__
+
+* `JDBC <https://github.com/aiven/aiven-kafka-connect-jdbc/blob/master/docs/source-connector.md>`__
+
+* Schema Source
+
+* `Stream Reactor Cassandra <https://docs.lenses.io/connectors/source/cassandra.html>`__
+
+* `Stream Reactor MQTT <https://docs.lenses.io/connectors/source/mqtt.html>`__
+
+Sink connectors
+-----------------
+
+Sink connectors enable the integration of data from an existing Apache Kafka topic to a target technology. The following is the list of available connectors:
```

Comment:
"list of available connectors:" I'd change. " list of available sink connectors:"

Reply:
fixed